src/Toolkit/Toolkit.Maui/SymbolDisplay/SymbolDisplay.cs

```diff
@@ -137,7 +152,6 @@ private async Task UpdateSwatchAsync()
         }
     }
 
-    // Known issue - see toolkit issue #451
     private double GetScaleFactor() => Window?.DisplayDensity ?? 1d;
```

Comment:
I wonder if this code could be called before we have a window, and then will be called again later. Of so, perhaps we should just return NaN here and not generate a swatch until the window gets established?